### PR TITLE
feat(SPDX): Upgrade tools-java library to support SPDX 2.3 format

### PR DESCRIPTION
**Pull request by tuannn2**
Signed-off-by: Nguyen Nhu Tuan <tuan2.nguyennhu@toshiba.co.jp>

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change?

Issue:  #1732

#### Import SPDX BOM in Project page
Use file 
[bom.spdx.rdf.zip](https://github.com/eclipse/sw360/files/10091067/bom.spdx.rdf.zip)

Steps
1. Go to project page
2. Click "Import SPDX BOM" button
3. Upload bom.spdx.rdf file

Validate
- "asoftware (1.1)" project was created
- "Browser", "Buffer" and "CompressionEng" Components were created
- "Browser(2.1)", "Buffer(2.2)" and "CompressionEng 3.1" Releases were created
- linked releases of "asoftware (1.1)" project are Browser(2.1) and Buffer(2.2)
- linked release of "Browser(2.1)" is "CompressionEng 3.1"

#### Import SPDX BOM in Component page
Use file 
[bom.spdx.rdf.zip](https://github.com/eclipse/sw360/files/10091071/bom.spdx.rdf.zip)

Steps
1. Go to component page
2. Click "Import SPDX BOM" button
3. Upload bom.spdx.rdf file

Validate
- "asoftware", "Browser", "Buffer" and "CompressionEng" Components were created
- "asoftware (1.1)", "Browser(2.1)", "Buffer(2.2)" and "CompressionEng 3.1" Releases were created
- linked releases of "asoftware (1.1)" release are Browser(2.1) and Buffer(2.2)
- linked release of "Browser(2.1)" is "CompressionEng 3.1"

#### Import SPDX BOM 2.3
Use file https://github.com/spdx/tools-java/blob/master/testResources/SPDXTagExample-v2.3.spdx

Steps
1. Go to component page
2. Click "Import SPDX BOM" button
3. Upload SPDXTagExample-v2.3.spdx file
4. 
Validate
- "glibc" Components were created
- "glibc 2.11.1" Releases were created

### Checklist
Must:
- [ ] All related issues are referenced in commit messages and in PR

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -362,7 +362,7 @@
             <dependency>
                 <groupId>org.spdx</groupId>
                 <artifactId>tools-java</artifactId>
-                <version>1.0.4</version>
+                <version>1.1.0</version>
```
Comment (heliocastro) [inline pom.xml:365]: Please add the version on the same way as the other dependency version on top of the file

**Review by heliocastro — CHANGES_REQUESTED** [review]
Please add the version like this on top:
```<org.spdx.tools.java.version>1.1.0</org.spdx.tools.java.version>```
And on the dependency entry:
```<version>${org.spdx.tools.java.version}</version>```

**Comment by tuannn2** [comment]
Thank you for your reviewing, @heliocastro  . I have fixed it according to your review. Please check it!

**Review by heliocastro — APPROVED** [review]
LGTM

**Pull request merged**